### PR TITLE
added changes on the docs with reference to latest changes on nginx ip

### PR DESCRIPTION
**Pull request by Cbkhare**
## Description

Details on resources on webroot and osie can be reached.

## Why is this needed

With reference to recent changes on nginx IP. This change was required.

### CODE REVIEW

File: docs/setup/using-qemu.md [hunk 1]
```diff
@@ -169,14 +169,19 @@ This installation method also requires Docker and [`docker-compose`](https://doc
...
     ```
-    # Add Nginx address to Tinkerbell
-    sudo ip addr add 192.168.1.2/24 dev tinkerbell
     cd deploy
     source ../.env; docker-compose up -d
     ```
 
     At this point we now have a server with available resource, we can create virtual machines and Tinkerbell is listening on the correct internal network!
 
+    Reasource inside the Tinkrebell will be reachable as per the below details
```
Comment (gauravgahlot) [inline docs/setup/using-qemu.md:178]:
```suggestion
    Resource inside the Tinkerbell will be reachable as per the below details
```

**Review by gauravgahlot — APPROVED** [review]
LGTM

**Comment by gauravgahlot** [comment]
@Cbkhare Could you update the branch, please?

**Review by thebsdbox — APPROVED** [review]
Excellent! looks good and fixes the confusion around the ports.

**Pull request merged**